doc/conf.py

```diff
@@ -31,20 +31,55 @@
 import sys
 import os
 
+# import sphinx_gallery
 import pip
 
 import subprocess
 
 from subprocess import check_output, CalledProcessError
+from mock import Mock as MagicMock
+
+source_suffix = ['.rst']
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('./..'))
-sys.path.insert(0, os.path.abspath('./../topology'))
-sys.path.insert(0, os.path.abspath('./../pynest/nest'))
 
-source_suffix = ['.rst']
+doc_path = os.path.abspath(os.path.dirname(__file__))
+root_path = os.path.abspath(doc_path + "/..")
+
+sys.path.insert(0, os.path.abspath(root_path))
+sys.path.insert(0, os.path.abspath(root_path + '/topology'))
+sys.path.insert(0, os.path.abspath(root_path + '/pynest/'))
+sys.path.insert(0, os.path.abspath(root_path + '/pynest/nest'))
+sys.path.insert(0, os.path.abspath(doc_path))
+
+
+# -- Mock pynestkernel ----------------------------------------------------
+# The mock_kernel has to be imported after setting the correct sys paths.
+from mock_kernel import convert  # noqa
```

Comment:
why not use the proper kernel if the install is not done on RtD?
If this really is a feature, then remove following comment lines

Reply:
We were thinking for consistency, just leaving the mock_kernel for non RtD builds too; it then allows the docs to build without a proper nest install - Not sure if that's a 'feature', but could be useful.

Reply:
OK, I don't have any preference, but if we keep the mock_kernel for all builds, then comments should go away ;)

Reply:
will do :)